filtering/parser.go

```diff
@@ -74,7 +74,7 @@ func (p *Parser) ParseExpression() (_ *expr.Expr, err error) {
 			err = p.wrapf(err, start, "expression")
 		}
 	}()
-	sequences := make([]*expr.Expr, 0)
+	sequences := make([]*expr.Expr, 0, 1)
```

Comment:
Hey @odsod,

I took a look at this and fixed up a gosec out of bounds lint warning. I'm 99% sure the change shouldn't have any side effects but haven't looked at this code before. What do you think?

Reply:
Nice LGTM